container/src/LuigiContainer.svelte

```diff
@@ -151,6 +152,13 @@
       webcomponentService.thisComponent = thisComponent;
 
       const ctx = GenericHelperFunctions.resolveContext(context);
+
+      thisComponent.updateViewUrl = (viewUrl: string, internal?: object) => {
+        if (viewUrl?.length) {
+          ContainerAPI.updateViewUrl(viewUrl, ctx, internal, iframeHandle);
```

Comment:
```suggestion
          ContainerAPI.updateViewUrl(viewUrl, GenericHelperFunctions.resolveContext(context), internal, iframeHandle);
```
Here the ctx is still the constant from init phase (se previous comment), so later context updates would be ignored. Therefore "live context" must be used.